src/marvin/beta/assistants/assistants.py

```diff
@@ -186,10 +187,25 @@ async def load_async(cls, assistant_id: str, **kwargs):
         assistant._context_level = 1
         return assistant
 
-    def chat(self, thread: Thread = None):
-        if thread is None:
-            thread = self.default_thread
-        return thread.chat(assistant=self)
+    @expose_sync_method("chat")
+    async def chat_async(self, initial_message: str = None, **kwargs):
+        # send an initial message, if provided
+        if initial_message is not None:
+            await self.say_async(initial_message, **kwargs)
+        while True:
+            try:
+                message = Prompt.ask("[bold green]Your message[/]")
+                # if the user types exit, ask for confirmation
+                if message == "exit":
```

Comment:
just a personal request :) 
```suggestion
                if message in ["exit", "!exit", ":q", "!quit"]:
```